WikiDrips/ImageGenerator.swift

```diff
@@ -8,8 +8,10 @@
 
 import Foundation
 import UIKit
+import os.log
 
-class ImageGenerator: Operation{
+class ImageGenerator: Operation {
+    static let ig_log = OSLog(subsystem: "com.salesforce.WikiDrips", category: "ImageGenerator")
```

Comment:
Whoops, rubyisms and underscores. Thanks for catching, @nartiles. 

Is there a pattern for when underscores are appropriate? I ask because os_log is a function defined in OSLog (https://developer.apple.com/documentation/os/2320718-os_log). Is that just an objective c leftover or something?